### DISCUSSION TITLE
AW Won't Launch (Windows)

### DISCUSSION THREAD

**Discussion by celiahein** [ActivityWatch/activitywatch#1046] [Q&A]
I installed the latest version for Windows using the standard installer, and it won't launch. In the Task Manager, I can see the application and its associated files, but they're all running at 0%. The AW never even shows up in the task bar. I've tried restarting the computer and running AW as an admin, with no luck. Am I missing some security permission or something? I'm on Windows 11 pro

**Comment by BelKed** [comment]
What version of ActvityWatch are you running?
There is a similar problem with the latest beta (`v0.12.3b16`), the one before should work fine (`v0.12.3b15`) :)
